rostran/cli/__main__.py

```diff
@@ -116,6 +118,24 @@ def generate(
     typer.echo(f'Generate "{resource_type}" to ROS template (Format: {file_format})')
 
 
+@app.command()
+def parse(
```

Comment:
This command should move to a new cmd file, e.g. tools/generate.py . And supports parse command with --terraform option.